core/broker/uplink.go

```diff
@@ -135,6 +150,13 @@ func (b *broker) HandleUplink(uplink *pb.UplinkMessage) (err error) {
 		"DevID":     device.DevId,
 		"FCnt":      originalFCnt,
 	})
+	deduplicatedUplink.DevEui = device.DevEui
+	deduplicatedUplink.AppEui = device.AppEui
+	deduplicatedUplink.AppId = device.AppId
+	deduplicatedUplink.DevId = device.DevId
+	deduplicatedUplink.Trace = deduplicatedUplink.Trace.WithEvent("mic checked",
```

Comment:
Why not using a const here? If it's not as common to promote it to the trace package, use a const in this package